src/topology/algebra/module/basic.lean

```diff
@@ -935,6 +935,20 @@ rfl
   ⇑(f₁.prod_map f₂) = prod.map f₁ f₂ :=
 rfl
 
+lemma fst_prod_zero_add_zero_prod_snd [has_continuous_add M₁] [has_continuous_add M'₁] :
+  (continuous_linear_map.fst R₁ M₁ M'₁).prod 0 +
```

Comment:
Isn't `inl` the normal form for this? This looks like `coprod_inl_inr` to me.

Reply:
Ok, I'll reformulate/rename to `continuous_linear_map.coprod_inl_inr`.